src/app/_components/OrbitEventsSection.tsx

```diff
@@ -0,0 +1,167 @@
+import Airtable from 'airtable'
+import z from 'zod'
+
+import { usePagination } from '@/hooks/usePagination'
+import { useSearch } from '@/hooks/useSearch'
+
+import { Button } from '@/components/Button'
+import { Card } from '@/components/Card'
+import { CardGrid } from '@/components/CardGrid'
+import { FilterContainer } from '@/components/FilterContainer'
+import { NoResultsMessage } from '@/components/NoResultsMessage'
+import { Pagination } from '@/components/Pagination'
+import { Search } from '@/components/Search'
+
+import { NextServerSearchParams } from '@/types/searchParams'
+
+import { formatDate } from '@/utils/formatDate'
+
+import { FILECOIN_FOUNDATION_URLS } from '@/constants/siteMetadata'
```

Comment:
```suggestion
import { FILECOIN_FOUNDATION_URLS } from '@/constants/siteMetadata'

import { fetchAndParseAirtableEvents } from './_api/airtable'

```

Reply:
I've added the airtable-related code into `orbit/services/airtable.ts`.

I didn't name it `api` because it used to be a reserved keyword, and it might imply that this file creates an API endpoint.

I hope it makes sense.

- https://nextjs.org/docs/app/building-your-application/routing/route-handlers
- https://nextjs.org/docs/pages/building-your-application/routing/api-routes